test/new_relic/agent/tracer_test.rb

```diff
@@ -400,7 +400,7 @@ def test_thread_ids_included_when_enabled
             Thread.new { 'woof' }.join
           end
 
-          assert_match /Ruby\/Thread\/Thread\d+\/Fiber\d+/, txn.segments.last.name
+          assert_match (/Ruby\/Thread\/Thread\d+\/Fiber\d+/), txn.segments.last.name
```

Comment:
In both of these cases, due to the fact that the patterns themselves need to reference literal forward slashes, I think we should have used `%r{}` in the first place to not have to use the backslashes for escaping. For example:

```ruby
assert_match %r{Ruby/Thread/Thread\d+/Fiber\d+}, txn.segments.last.name
```

Reply:
Great suggestion! I updated it to that in 4f4b507be7f803fcd10b2de3773e81ebe03b9e04